### PR TITLE
add initial readme for troubleshooting CI integration tests failures

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: playwright/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## Troubleshooting Failed Integration Tests in CI
+
+### Single Test Failure
+
+There are stored artifacts in the GitHub CI. Go to the failed job and there is
+a link "Summary". At the bottom of the page there is "Artifacts" section which
+contains the `y2log` and also `trace.zip` file. The trace can be browsed using
+the playwright too locally or at page https://trace.playwright.dev/ to get
```
Comment (imobachgs) [inline playwright/README.md:8]: tool?

File: playwright/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+details of the failure.
+
+### Stuck at DBus Loading
+
+It usually indicates an issue with the DInstaller DBus services. There is a step
```
Comment (imobachgs) [inline playwright/README.md:13]:
```suggestion
It usually indicates an issue with the D-Installer D-Bus services. There is a step
```

File: playwright/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+### Single Test Failure
+
+There are stored artifacts in the GitHub CI. Go to the failed job and there is
+a link "Summary". At the bottom of the page there is "Artifacts" section which
+contains the `y2log` and also `trace.zip` file. The trace can be browsed using
+the playwright too locally or at page https://trace.playwright.dev/ to get
+details of the failure.
+
+### Stuck at DBus Loading
```
Comment (imobachgs) [inline playwright/README.md:11]:
```suggestion
### Stuck at D-Bus Loading
```

File: playwright/README.md (hunk 1 above)
Comment (jreidinger) [reply playwright/README.md]: probably. @lslezak you add it. So if you can confirm it

**Review by imobachgs — APPROVED** [review]

**Pull request merged**